### PR TITLE
Fix #35 - disable debug mode by default.

### LINKED ISSUE #35: Default enabling of __MODBUSMASTER_DEBUG__ leads to unpredictable behavior in sketches, suggest setting to 0 by default
First off, great library and thanks for making it, overall I've been very happy with it and it has worked perfectly for my project. 

However, these few lines were causing some odd behavior in my system that it took me a while to track down. 

````ModbusMaster.cpp lines 150-153:````
````
#if __MODBUSMASTER_DEBUG__
  pinMode(4, OUTPUT);
  pinMode(5, OUTPUT);
#endif
````

The way I constructed my application is part of the reason it took a while, but generally I don't expect a library to change the state of pins in a hidden way. I recommend setting __MODBUSMASTER_DEBUG__ to 0 by default, since if you intend to use the debugging features you will have no issue in enabling them, but most users will probably not be aware of this functionality and are more likely to be using digital pins 4 and 5. 

If nothing else, it would be good to make the fact that this is enabled by default very obvious and mention it in the main README. 

Thanks again for the library.

### PR DESCRIPTION
**Pull request by kintel**
Also made debug pins configurable.

### CODE REVIEW

**Comment by 4-20ma** [comment]
Interested in the PR. I'll review and test out.

File: ModbusMaster.cpp [hunk 1]
```diff
@@ -148,8 +148,8 @@ void ModbusMaster::begin(uint16_t u16BaudRate)
   
   MBSerial->begin(u16BaudRate);
 #if __MODBUSMASTER_DEBUG__
-  pinMode(4, OUTPUT);
-  pinMode(5, OUTPUT);
+  pinMode(__MODBUSMASTER_DEBUG_PIN_A__, OUTPUT);
+  pinMode__MODBUSMASTER_DEBUG_PIN_B__, OUTPUT);
```
Comment (4-20ma) [inline ModbusMaster.cpp]: Appears to be missing `(`.

File: ModbusMaster.cpp [hunk 2]
```diff
@@ -772,25 +772,25 @@ uint8_t ModbusMaster::ModbusMasterTransaction(uint8_t u8MBFunction)
...
 #if __MODBUSMASTER_DEBUG__
-      digitalWrite(4, false);
+      digitalWrite(__MODBUSMASTER_DEBUG_PIN_A__, false);
 #endif
     }
     else
     {
 #if __MODBUSMASTER_DEBUG__
-      digitalWrite(5, true);
+      digitalWrite__MODBUSMASTER_DEBUG_PIN_B__, true);
```
Comment (4-20ma) [inline ModbusMaster.cpp]: Appears to be missing `(`.

File: ModbusMaster.cpp [hunk 3]
```diff
@@ -772,25 +772,25 @@ uint8_t ModbusMaster::ModbusMasterTransaction(uint8_t u8MBFunction)
...
 #endif
       if (_idle)
       {
         _idle();
       }
 #if __MODBUSMASTER_DEBUG__
-      digitalWrite(5, false);
+      digitalWrite__MODBUSMASTER_DEBUG_PIN_B__, false);
```
Comment (4-20ma) [inline ModbusMaster.cpp]: Appears to be missing `(`.

**Comment by 4-20ma** [comment]
Please verify missing `(` and resolve. Thanks.

**Comment by kintel** [comment]
Forgot to test with debug turned on :/
Merged with master and fixed missing '('.

**Comment by 4-20ma** [comment]
LGTM.

**Pull request merged**